ext/ffi/tests/ffi_load_error.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+Test that FFI::load returns a detailed error when failling to load a shared library
+--EXTENSIONS--
+ffi
+--INI--
+ffi.enable=1
+--FILE--
+<?php
+FFI::load(__DIR__ . "/ffi_load_error.h");
+?>
+--EXPECTF--
+Fatal error: Uncaught FFI\Exception: Failed loading '%s' (%s) in %s:%d
+Stack trace:
```

Comment:
I don't think this stack backtrace is particularly useful, so I suggest to catch the `FFI\Exception`, and to only prints its `::getMessage()` result.

Reply:
It is now changed, sorry for the delay.